tdrs-backend/tdpservice/data_files/test/test_stuck_files.py

```diff
@@ -0,0 +1,406 @@
+"""Test the get_stuck_files function."""
+
+
+import pytest
+from datetime import datetime, timedelta
+from django.utils import timezone
+from tdpservice.data_files.models import DataFile
+from tdpservice.parsers.models import DataFileSummary
+from tdpservice.search_indexes.models.reparse_meta import ReparseMeta
+from tdpservice.data_files.tasks import get_stuck_files
+
+
+def _time_ago(hours=0, minutes=0, seconds=0):
+    return datetime.now(tz=timezone.utc) - timedelta(hours=hours, minutes=minutes, seconds=seconds)
+
+
+@pytest.mark.django_db
+def test_find_pending_submissions__none_stuck(stt_user, stt):
+    """Finds no stuck files."""
+    df1 = DataFile.objects.create(
+        quarter=DataFile.Quarter.Q1, section=DataFile.Section.ACTIVE_CASE_DATA,
+        year=2023, version=1, user=stt_user, stt=stt
+    )
+    df1.created_at = _time_ago(hours=2)
+    df1.save()
+
+    _ = DataFileSummary.objects.create(
```

Comment:
Does the linter complain about not having a variable to catch the created object? If not, can we remove the captures?